hera_cal/nucal.py

```diff
@@ -708,14 +708,27 @@ def _linear_fit(XTX, Xy, solver='lu_solve', tol=1e-15):
         tol : float, default=1e-15
             Tolerance to use for regularization. If method is 'lu_solve' or 'solve', this is added to the diagonal of XTX. 
             If method is 'pinv' or 'lstsq', this is used as the rcond parameter for np.linalg.pinv and np.linalg.lstsq respectively.
+        cached_input : dictionary, default={}
```

Comment:
I'd explicitly reference the text for cached_output to describe the format this should take.